README.md

```diff
@@ -47,23 +46,35 @@ To make the `fh` CLI readily available on a NixOS system:
 ### Add a flake published to FlakeHub to your `flake.nix`
 
 `fh add` adds the most current release of the specified flake to your `flake.nix` and updates the `outputs` function to accept it.
+This would add the current release of [Nixpkgs] to your flake:
 
-```console
-$ fh add nixos/nixpkgs
+```shell
+fh add nixos/nixpkgs
+cat flake.nix
+```
 
-$ cat flake.nix
+```nix
```

Comment:
nit:

very tiny -1 to this separation, I don't think the nix syntax highlighting is worth more than the fake-terminal-flow presented originally, and I doubt this is a very high-traffic copy-paste area to warrant making it easier to copy-paste directly into a terminal.

Reply:
Fair enough